js/addToTask.js

```diff
@@ -0,0 +1,74 @@
+import {
+  createTaskButton,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+export let LIST = [];
+export let id = 0;
+let tarikh;
+function addToDo(toDo, id, done, edit, trash) {
+  if (trash) {
+    return;
+  }
+
+  //Presenting Todays date in task list inside li tag
+  const options = {
+    day: "numeric",
+    month: "numeric",
+    year: "numeric",
+  };
+
+  tarikh = today.toLocaleDateString("en-us", options);
+  const item = `<li class="item">                  
+                  <p class="text">${toDo}</p>
+                  <!-- data presenter div tag with date class  -->
+                  <div class = "date" id = "${id}"> Created At:  ${tarikh} </div>
```

Comment:
Can we consider using a template literal for the date string: Instead of using string concatenation to build the date string? It will make the code more concise and easier to read

Reply:
it's already in the template literal starting upon <li> tag